cmd/openshift-install/testdata/agent/agentconfigtemplate/agent-config-template.txt

```diff
@@ -22,6 +22,7 @@ metadata:
 # All fields are optional
 rendezvousIP: your-node0-ip
 bootArtifactsBaseURL: http://user-specified-infra.com
+minimalISO: false
```

Comment:
IIRC did we discuss also to add a case with `minimalISO: true` (and check a threshold size)?

Reply:
Yes, that is tracked in https://issues.redhat.com/browse/AGENT-974 and will be a separate PR.